src/openms/source/FORMAT/PercolatorInfile.cpp

```diff
@@ -174,13 +264,16 @@ namespace OpenMS
         pids.back().setScoreType(score_name);
         pids.back().setMetaValue(Constants::UserParam::ID_MERGE_INDEX, map_filename_to_idx.at(raw_file_name));
         pids.back().setRT(row[to_idx.at("retentiontime")].toDouble() * 60.0); // search engines typically write minutes (e.g., sage)
-        pids.back().setMetaValue("PinSpecId", sSpecId);
+        pids.back().setMetaValue("PinSpecId", sSpecId);  
+        if(SageAnnotation)
```

Comment:
```suggestion
        if (SageAnnotation)
```

Reply:
Applied spacing to all if-statements, thanks for the reminder!